ui/src/components/Header.tsx

```diff
@@ -1,12 +1,11 @@
-import { Fragment, useCallback } from "react";
+import { useCallback } from "react";
 import { useNavigate } from "react-router-dom";
 import { ArrowLeftEndOnRectangleIcon, ChevronDownIcon } from "@heroicons/react/16/solid";
-import { Menu, MenuButton } from "@headlessui/react";
+import { Button, Menu, MenuButton, MenuItem, MenuItems } from "@headlessui/react";
```

Comment:
Restructured due to the new Menu code. Using the headlessui's version of `Button` because it works more reliably with their `MenuItem`(s)